testgrid/README.md

```diff
@@ -402,43 +101,17 @@ Under **Options**
     vice versa) with more weight given to more recent transitions.
 *   **Sort by Name**: Sort alphabetically.
 
-## Unit testing
-
-Run `bazel test //testgrid/...` to ensure the config is valid.
```

Comment:
nit: not clear to me whether this was intentionally deleted, I feel like it belongs in build_test_update.md

Reply:
It was moved to the configuration.md file; under #testing-your-configuration.

It seemed that someone changing a configuration who might not be familiar with "bazel test" at all would need to know about this test function.

Reply:
got it; at present I'd still prefer it in build_test_update, the configuration file seems like a reference doc, not a how-to/playbook, but if others feel strongly the other way I'll let it drop

Reply:
I included a Testing section that lists the command here, but doesn't go into detail into its effects on config.yaml so that the doc isn't repeating itself.

Hopefully the best of both worlds